cmsplugin_offices_ai/models/plugin_models.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -:-
+from cms.models.pluginmodel import CMSPlugin
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+from cmsplugin_offices_ai.models import Department, Office, Person
+
+
+@python_2_unicode_compatible
+class DepartmentListPlugin(CMSPlugin):
+    """
+    Plugin model for displaying selected departments.
+    """
+    departments = models.ManyToManyField(
+        Department,
+        verbose_name=_('department'),
+        related_name='department_list_plugins',
+    )
+    show_personnel = models.BooleanField(
+        verbose_name=_('show personnel'),
+        default=False,
+        help_text=_('Show personnel belonging to each department.')
+    )
+
+    def __str__(self):
+        count = self.departments.all().count()
+        if count == 1:
+            return _('Department (%s)') % self.departments.first().name
+        return _('Departments (amount: %s)') % count
+
+    def copy_relations(self, old_instance):
```

Comment:
Actually there's no point in moving this to base class. We would still have the need to override `copy_relations` for each plugin model so we could call some helper function which basically does the same assignment with `getattr` and `setattr`. Yeah the comment in each is basically copy-pasted but I think it's good to have. Someone might not know Django CMS well enough to understand why the function is needed.